Shopify/cli
# Fix Banner potentially crashing on very narrow terminals

### WHY are these changes introduced?

Fixes https://github.com/Shopify/internal-cli-foundations/issues/549

### WHAT is this pull request doing?

`repeat` throws an error if the argument is negative. This PR makes sure that doesn't happen.

### How to test your changes?

- Make your terminal very narrow, around 5 columns should do it.
- Run `deploy`
- It shouldn't crash

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
